nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/nifi.properties

```diff
@@ -131,8 +131,10 @@ nifi.web.jetty.threads=${nifi.web.jetty.threads}
 
 # security properties #
 nifi.sensitive.props.key=
+nifi.sensitive.props.key.protected=${nifi.sensitive.props.key.protected}
```

Comment:
After building this PR, in conf/nifi.properties the property nifi.sensitive.props.key.protected was empty. I then filled in a value for nifi.sensitive.props.key and when I ran the toolkit it did not protect nifi.sensitive.props.key, I assume because it saw nifi.sensitive.props.key.protected was already there? I tested it again by first deleting the empty protected property and that time it did protect the sensitive properties key.

Seems like nifi.sensitive.props.key.protected should not be here by default, or the toolkit should recognize an empty value and protect it.